### ISSUE TITLE
Flow errors from imported npm package in buildAstSchema

### ISSUE THREAD

**Issue by TxHawks** [graphql/graphql-js#1586]
Using graphql-js version 14.0.2 and Flow version 0.87.0, Flow throws the following errors:

```sh
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/graphql/utilities/buildASTSchema.js.flow:363:9

Missing type annotation for V. V is a type parameter declared in function [1] and was implicitly
instantiated at call of keyValMap [2].

     node_modules/graphql/utilities/buildASTSchema.js.flow
     360│     def: ObjectTypeDefinitionNode | InterfaceTypeDefinitionNode,
     361│   ) {
     362│     return def.fields
 [2] 363│       ? keyValMap(
     364│           def.fields,
     365│           field => field.name.value,
     366│           field => this.buildField(field),
     367│         )
     368│       : {};
     369│   }
     370│

     node_modules/graphql/jsutils/keyValMap.js.flow
 [1]  29│ export default function keyValMap<T, V>(
      30│   list: $ReadOnlyArray<T>,
      31│   keyFn: (item: T) => string,
      32│   valFn: (item: T) => V,
      33│ ): ObjMap<V> {


Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/graphql/utilities/buildASTSchema.js.flow:372:12

Missing type annotation for V. V is a type parameter declared in function [1] and was implicitly
instantiated at call of keyValMap [2].

     node_modules/graphql/utilities/buildASTSchema.js.flow
     369│   }
     370│
     371│   _makeInputValues(values: $ReadOnlyArray<InputValueDefinitionNode>) {
 [2] 372│     return keyValMap(
     373│       values,
     374│       value => value.name.value,
     375│       value => this.buildInputField(value),
     376│     );
     377│   }
     378│
     379│   _makeInterfaceDef(def: InterfaceTypeDefinitionNode) {

     node_modules/graphql/jsutils/keyValMap.js.flow
 [1]  29│ export default function keyValMap<T, V>(
      30│   list: $ReadOnlyArray<T>,
      31│   keyFn: (item: T) => string,
      32│   valFn: (item: T) => V,
      33│ ): ObjMap<V> {


Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ node_modules/graphql/utilities/buildASTSchema.js.flow:399:9

Missing type annotation for V. V is a type parameter declared in function [1] and was implicitly
instantiated at call of keyValMap [2].

     node_modules/graphql/utilities/buildASTSchema.js.flow
     396│
     397│   _makeValueDefMap(def: EnumTypeDefinitionNode) {
     398│     return def.values
 [2] 399│       ? keyValMap(
     400│           def.values,
     401│           enumValue => enumValue.name.value,
     402│           enumValue => this.buildEnumValue(enumValue),
     403│         )
     404│       : {};
     405│   }
     406│

     node_modules/graphql/jsutils/keyValMap.js.flow
 [1]  29│ export default function keyValMap<T, V>(
      30│   list: $ReadOnlyArray<T>,
      31│   keyFn: (item: T) => string,
      32│   valFn: (item: T) => V,
      33│ ): ObjMap<V> {
```

As a quick fix, I've added `.*/node_modules/graphql/jsutils/keyValMap.js.flow` to my `.flowconfig`, but this is far from ideal...

**Comment by alizhdanov** [comment]
The same for me, thanks for the workaround. I think it's already fixed, but not yet released - https://github.com/graphql/graphql-js/commit/0ea7b5192cd3e3f55352d276ece87a83add7c68b

**Comment by IvanGoncharov** [comment]
> The same for me, thanks for the workaround. I think it's already fixed, but not yet released - 0ea7b51

@alizhdanov Correct, you can test using this package:
https://github.com/graphql/graphql-js#want-to-ride-the-bleeding-edge

@mjmahone Can we release `14.1.0` as is to publish fix for this issue and #1518 ?
I think we can push all other planned items to `14.2.0`.
What do you think?

**Comment by mjmahone** [comment]
I'd be happy pushing 14.2.0 this week. I'll try to ping @leebyron to make sure he's aware.

**Comment by IvanGoncharov** [comment]
@mjmahone @leebyron It would be great to update tokens inside:
https://github.com/graphql/graphql-js/blob/97568af33516f189e50da7e8234f5799d4b4121b/.travis.yml#L44-L51
That way we will be able to do automatic releases and release bugfixes faster.

**Comment by wincent** [comment]
> That way we will be able to do automatic releases and release bugfixes faster.

That would be nice. :-)

**Comment by mgrip** [comment]
Seems like .89 was releases but this is still broken?

**Comment by IvanGoncharov** [comment]
@mjmahone @leebyron Can you please update NPM token so we can release RC?

Currently we block Flow users from updating Flow.
More recent example:
https://github.com/graphql/graphql-js/commit/4cdc8e2a283e5e551cc462fbd00b0ce01c9126af#commitcomment-31933578
> Could you please release this change so we are not blocked on it? 🙂

**Comment by IvanGoncharov** [comment]
@mjmahone just published `14.1.0` so this issue should be fully resolved.

**Comment by wincent** [comment]
And I see 14.1.1 fast-followed. Thanks!
